crates/bevy_winit/src/winit_monitors.rs

```diff
@@ -0,0 +1,27 @@
+use winit::monitor::MonitorHandle;
+
+use bevy_ecs::entity::Entity;
+use bevy_ecs::system::Resource;
+
+/// Stores [`winit`] monitors and their corresponding entities
+#[derive(Resource, Debug, Default)]
```

Comment:
Any value in this being `Reflect`?

Reply:
Resources being reflect is mildly useful, but IME not nearly as important as for things that want to be inspected or serialized.